examples/msan_fuzz_test.cc

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// A fuzz target that causes an MSAN error (e.g., uninitialized variables).
+
+#include <cstdint>
+#include <cstddef>
+
+int DoMsan() {
```

Comment:
Use a more specific name here that describes what this method does. Please read this section, this will be useful for Starlark code too: https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Maybe call this `TriggerMsanError`?